src/services/TemplatesService.js

```diff
@@ -0,0 +1,26 @@
+import { api } from 'boot/axios';
+import { getTemplatesByType } from 'src/composables/TemplateManager';
+
+/**
+ * @param {string} type - project | component | model
+ * @returns {Promise<object>}
```

Comment:
rework on documentation.
Missing default description, description param are no explain of what is the param
Return have a break line and incomplete description.

Reply:
Done

Reply:
PROJECT | COMPONENT | MODEL -> `, can be 'PROJECT', 'COMPONENT' or 'MODEL'.`